src/main/kotlin/skytils/skytilsmod/core/Config.kt

```diff
@@ -1896,6 +1896,13 @@ object Config : Vigilant(
     )
     var rareRelicFinder = false
 
+    @Property(
```

Comment:
why did you pick an arbitrary location

Reply:
i put it with the other misc features

Reply:
you put it in between a feature and a button that goes with the feature